virtcontainers/kata_agent.go

```diff
@@ -54,6 +54,8 @@ const (
 
 	// path to vfio devices
 	vfioPath = "/dev/vfio/"
+
+	agentPidEnv = "KATA_AGENT_PIDNS"
```

Comment:
Could you use container annotations instead of environment variables to enable it? It turns into a container infra visible thing but not present to the actual running processes.

Reply:
nvm, I just realized that annotations are pod level resources.